### DISCUSSION TITLE
User input on html page

### DISCUSSION THREAD

**Discussion by smitpi** [EvotecIT/PSWriteHTML#217] [Q&A] [answered]
Hi all.

Is there a way I can get a user to import data? 

Like a textbox with a button, and action something then the button is clicked?

Thanks for the help.

**Comment by PrzemyslawKlys** [comment, marked as answer]
Not at the moment. It would require page to be hosted and I haven't done any tests around it.

**Comment by smitpi** [comment]
okay thanks for the feedback.

Any suggestions you can think of, that might help?

**Reply by PrzemyslawKlys** [reply]
Universal Dashboard.

**Reply by endoleg** [reply]
PodeWeb?
